valohai_yaml/objs/pipelines/execution_node.py

```diff
@@ -41,12 +42,20 @@ def lint(self, lint_result: LintResult, context: LintContext) -> None:
         if self.override is not None:
             self.override.lint(lint_result, context)
             step_inputs = step.inputs.keys()
+            strictly_non_replaceable_inputs = [
+                inp.name for inp in step.inputs.values() if inp.override_mode == InputOverrideMode.APPEND.value
+            ]
```

Comment:
I'm not sure this makes sense, _if_ the end goal is that pipeline edges can either replace or append inputs?

Reply:
if someone choose append here 
and one pipeline edge defines explicit override (which means it would replace) 
its the conflicting thing, how should we tackle this then 🤔 or we should ignore it assuming replace has upper hand